config_example.sh

```diff
@@ -462,6 +462,16 @@ set -x
 #
 #export ENABLE_ARBITER=1
 
+# ENABLE_WORKLOAD_PARTITIONING -
+# Set to any non zero length string value to enable workload partitioning in the install config.
+#
+# Workload partitioning allows for workloads to be pinned to specific CPUSets with Performance Profiles
```

Comment:
So this setting can only be added when using arbiter node and the num_master is set to 2? If so might want to make that apparent in this description.